data/ha/hawk_test.py

```diff
@@ -0,0 +1,829 @@
+        test_click_on(browser, 'Resource', timeout_scale)
+        check_and_click_by_xpath(browser, timeout_scale, 12, "on Create Clone [%s]" % clone,
+                                 [CLONE_DATA_HELP_FILTER])
+        elem = find_element(browser, By.NAME, 'clone[id]')
+        if not elem:
+            print("ERROR: Couldn't find element [clone[id]]. No text-field where to type clone id")
+            RETURN_VALUE = 12
+            return
+        elem.send_keys(str(clone))
+        time.sleep(timeout_scale)
+        check_and_click_by_xpath(browser, timeout_scale, 12, "while adding clone [%s]" % clone,
+                                 [CLONE_CHILD, OPT_STONITH, TARGET_ROLE_FORMAT % 'clone',
+                                  TARGET_ROLE_STARTED, RSC_OK_SUBMIT])
+        if verify_success(browser):
+            print("INFO: Successfully added clone [%s] of [stonith-sbd]" % clone)
+            set_test_status('add_clone', 'passed')
+        else:
+            print("ERROR: Could not create clone [%s]" % clone)
+            RETURN_VALUE = 12
+
+def add_group(browser, groupname, timeout_scale):
+    global RETURN_VALUE
+    timeout_scale = checkint(timeout_scale)
+    test_click_on(browser, 'Resource', timeout_scale)
+    print("TEST: Adding group [%s]" % groupname)
+    check_and_click_by_xpath(browser, timeout_scale, 15, "while adding group [%s]" % groupname,
+                             [GROUP_DATA_FILTER])
+    if is_driver(browser):
+        elem = find_element(browser, By.NAME, 'group[id]')
+        if not elem:
+            print("ERROR: Couldn't find text-field [group[id]] to input group id")
+            RETURN_VALUE = 15
+            return
+        elem.send_keys(str(groupname))
+        time.sleep(timeout_scale)
+        check_and_click_by_xpath(browser, timeout_scale, 15, "while adding group [%s]" % groupname,
+                                 [STONITH_CHKBOX, TARGET_ROLE_FORMAT % 'group', TARGET_ROLE_STARTED,
+                                  RSC_OK_SUBMIT])
+        if verify_success(browser):
+            print("INFO: Successfully added group [%s] of [stonith-sbd]" % groupname)
+            set_test_status('add_group', 'passed')
+        else:
+            print("ERROR: Could not create group [%s]" % groupname)
+            RETURN_VALUE = 15
+
+def click_around_edit_conf(browser, timeout_scale):
+    print("TEST: Checking around Edit Configuration")
+    print("TEST: Will click on Constraints, Nodes, Tags, Alerts and Fencing")
+    check_and_click_by_xpath(browser, timeout_scale, 17,
+                             "while checking around edit configuration",
+                             [HREF_CONSTRAINTS, HREF_NODES, HREF_TAGS, HREF_ALERTS, HREF_FENCING])
+    if RETURN_VALUE != 17:
+        set_test_status('click_around_edit_conf', 'passed')
+
+# Set STONITH/sbd in maintenance. Assumes stonith-sbd resource is the last one listed on the
+# resources table
+def set_stonith_maintenance(browser, timeout_scale):
```

Comment:
would be nice to come up with some naming pattern , like all test cases end up or starting from some word